libred/CMakeLists.txt

```diff
@@ -16,6 +16,32 @@ check_symbol_exists(_NSGetEnviron crt_externs.h HAVE_NSGETENVIRON)
 
 find_package(Threads REQUIRED)
 
```

Comment:
Couldn't you just use  `#cmakedefine BEAR_TOOL "@BEAR_TOOL@"` to avoid duplicating the default values?